pkg/macvlan/macvlan.go

```diff
@@ -601,3 +629,7 @@ func CmdDel(args *skel.CmdArgs) error {
 
 	return nil
 }
+
+func isIPv6CIDR(cidr *net.IPNet) bool {
+	return cidr.IP.To4() == nil
```

Comment:
Might be worth doing `cidr.IP != nil && cidr.IP.To4() == nil` just to be on the safe side.